### PR TITLE
Tag Cliffords v0.4.0 [https://github.com/BBN-Q/Cliffords.jl]

### PR DESCRIPTION
**Pull request by blakejohnson**

### CODE REVIEW

**Comment by blakejohnson** [comment]
I may need to move the tag, so please hold off on merging for now.

**Comment by blakejohnson** [comment]
Tag moved. This should be good to go.

File: Cliffords/versions/0.4.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5-
```
Comment (tkelman) [inline Cliffords/versions/0.4.0/requires]: do you still need to support prereleases?

Comment (blakejohnson) [inline Cliffords/versions/0.4.0/requires]: No, I don't.

File: Cliffords/versions/0.4.0/requires [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Iterators
+Compat
```
Comment (blakejohnson) [inline Cliffords/versions/0.4.0/requires]: Ok, will do.

**Comment by blakejohnson** [comment]
This push should address the review comments.

**Comment by tkelman** [comment]
you're still `using Compat` in a few places in the tests, should those be removed or should it be added to `test/REQUIRE` ?

**Comment by blakejohnson** [comment]
The tests shouldn't need `Compat`, I think I just forgot to remove that line.

**Comment by tkelman** [comment]
there's one in `runtests.jl` and one in `rb.jl`

**Comment by blakejohnson** [comment]
Ok, thanks. I'll clean that up later this morning.

**Comment by blakejohnson** [comment]
This is ready for another look when you have a moment, @tkelman.

**Pull request merged**